src/utils/signature.ts

```diff
@@ -0,0 +1,25 @@
+import { verifyMessage } from '@ambire/signature-validator'
+import { JsonRpcSigner } from '@ethersproject/providers'
+import { readProvider } from 'constants/readProvider'
+
+export const verifyWallet = async (signer: JsonRpcSigner, message: string) => {
+  if (!signer || !message) {
+    return
+  }
+  const signature = await signer.signMessage(message)
+  return signature
+}
+
+export const verifySignature = async (
```

Comment:
**question** similar to above perhaps `verifyMessage` might be more clear?

Reply:
truuuu i guess there's basically no point to these utilities now that i look at them given they're such thin wrappers. maybe a future action item is to remove the ambire library as a dep and move its code into our codebase.

i just deleted the file for now.